priv/repo/migrations/20170424091310_soft_delete_user.exs

```diff
@@ -0,0 +1,9 @@
+defmodule CoursePlanner.Repo.Migrations.SoftDeleteUser do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :deleted_at, :utc_datetime
```

Comment:
One issue though, coherence has utc_datetime types.
The effort to keep it consistent should be higher for naive_datetime.